include/swift/Serialization/InlinableText.h

```diff
@@ -13,7 +13,9 @@
 #ifndef SWIFT_SERIALIZATION_INLINABLETEXT_H
 #define SWIFT_SERIALIZATION_INLINABLETEXT_H
 
+#include "swift/AST/Decl.h"
```

Comment:
I think the `class ParamDecl` would be sufficient for this one. It's `swift/Basic/LLVM.h` that you're still missing.

(Not that there's really a problem including Decl.h here, but then you don't need the forward declaration.)

Reply:
We should minimize header includes in headers as much as possible, to reduce compile time when you change something in a header (but so much depends on Decl.h that it won't help much in this particular case).

Reply:
Yeah, I’m gonna go ahead and revert back to the forward declaration.